api/URLSearchParams.json

```diff
@@ -896,7 +955,7 @@
               "version_added": false
             },
             "nodejs": {
-              "version_added": "7.5.0"
+              "version_added": "6.13.0"
```

Comment:
Same as here, `@@iterator` seems to have only been added in 7.3.0.

See: https://github.com/nodejs/node/commit/61d6293033782a2abfbb376c38d5d7c600be3e8d#diff-26e6b1e2a90040b1ce20e7f305b2ff3140b483f863ee2573b4c6af9b696ad31eR724

Reply:
v7.0.0 support is added here https://github.com/nodejs/node/pull/7448/commits/eaffcf9f1ac98247df44a4f8cf0a0f175b40183e#diff-26e6b1e2a90040b1ce20e7f305b2ff3140b483f863ee2573b4c6af9b696ad31eR569-R580

per https://github.com/nodejs/node/issues/55806#issuecomment-2468591380, the backlog to v6.13.0 is done via https://github.com/nodejs/node/pull/17365, though not mentioned in the release note

![image](https://github.com/user-attachments/assets/ddd69cb9-038e-4705-8112-4eab85a09e8e)